moveit_ros/moveit_servo/src/pose_tracking.cpp

```diff
@@ -235,7 +233,7 @@ void PoseTracking::targetPoseCallback(const geometry_msgs::msg::PoseStamped::Con
     try
     {
       geometry_msgs::msg::TransformStamped target_to_planning_frame = transform_buffer_.lookupTransform(
-          planning_frame_, target_pose_.header.frame_id, rclcpp::Time(0), rclcpp::Duration(0.1));
```

Comment:
Are you sure you want no timeout on this?  I'm not sure what that does but I imagine that causes it to block forever or return imidiatly.  I would assume what you want is something of a timeout (as there was before).

Reply:
Changed it to 100ms